### PR TITLE
Add package build target

### PR DESCRIPTION
**Pull request by matsievskiysv**
- [x] Branch rebased on latest master `git pull --rebase upstream master`
- [x] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

Add cmake build target `package` for creating binary packages (e.g. deb archives) of the compiled FreeCAD.
Useful for developer version installation.

### CODE REVIEW

File: package/debian/summary [hunk 1]
```diff
@@ -0,0 +1,10 @@
+ FreeCAD is an Open Source CAx RAD based on OpenCasCade, Qt and Python.
+ It features some key concepts like macro recording, workbenches, ability
+ to run as a server and dynamically loadable application extensions and
+ it is designed to be platform independent.
+ .
+ Currently, FreeCAD can import and display CAD models in IGES, STEP, and
+ BRep formats and meshes in STL, BMS, AST and Wavefront OBJ formats.
+ Editing and modeling features are currently somewhat limited.
```
Comment (vocx-fc) [inline package/debian/summary:8]: Where did you get this text from? This is clearly outdated information. I remember there was updated information in some other package.

Comment (matsievskiysv) [reply package/debian/summary:8]: @vocx-fc https://packages.debian.org/sid/freecad

Comment (matsievskiysv) [reply package/debian/summary]: Updated description.
This is a blend of deb and wiki descriptions.

**Comment by matsievskiysv** [comment]
Deb part works fine now.

**Pull request closed without merging**